snuba/datasets/entities/generic_metrics.py

```diff
@@ -110,7 +110,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             TagsTypeTransformer(),
             MappedGranularityProcessor(
                 accepted_granularities=PERFORMANCE_GRANULARITIES,
-                default_granularity_enum=DEFAULT_MAPPED_GRANULARITY_ENUM,
+                default_granularity=DEFAULT_MAPPED_GRANULARITY_ENUM,
```

Comment:
any reason for this change?

Reply:
I thought it made the YAML definition look cleaner